### DISCUSSION TITLE
New one

### DISCUSSION THREAD

**Discussion by Abumd98** [community/community#41827] [Repositories]
How to add repositories

**Comment by DSDmark** [comment]
Hi @Abumd98,
   
> Nice to meet you and welcome to this great community, we are here to help each other and try to make open source better. This is the discussions section if you want to discuss any. you can ask here, we will try to help.

And also, why any chance you are curious about, 

> what is GitHub, who is GitHub then I highly suggest you check this out [**What is GitHub?**](https://youtu.be/pBy1zgt0XPc).  its a video. so, just ***sit back and relax***

About the **GitHub** in short way,

> it's a place for anyone from anywhere, to build anything. **anythings** you think the world needs. 

![image](https://user-images.githubusercontent.com/85744816/206890536-5f5d663a-9284-4a0a-b189-8f244254bbac.png)

**Reply by Abumd98** [reply]
Thank u 👍

**Comment by DSDmark** [comment]
And about your question:-

The simple way to add repository on GitHub.

   - Sign in to your GitHub account and navigate to the GitHub homepage.

  -  Click on the "+" icon in the top right corner of the page, and then select "New repository" from the dropdown menu.

  -  In the "Repository name" field, enter a name for your repository.

  -  In the "Description" field, enter a brief description of your repository.

  -  Select whether you want to make the repository public or private.

   - Click the "Create repository" button to create the repository.

   - Once the repository is created, you can clone the repository to your local machine using the "Clone or download" button on the repository page, and then push your local code changes to the repository using the git command-line tool.
